docs/modules/ROOT/pages/references/service-maturity.adoc

```diff
@@ -0,0 +1,142 @@
+
+* access a metrics dashboard to get an overview of the Cloud Kubernetes environment and health.
+
+As a *Technical VSHNeer* I can:
+
+* have a metrics service that will help me keep track of the health and performance of the Cloud Kubernetes environment.
+* enable alerting on pre-defined SLIs to VSHN so that VSHN can resolve incidents with priority.
+* see SLO reports so that I can get an overview of how well my Cloud Kubernetes environment performs.
+* access operations runbooks so that I can resolve upcoming alerts.
+* access a metrics dashboard to get an overview of the Cloud Kubernetes environment and health.
+
+== Iteration: End-User Metrics and Alerting
+
+As an *End-User* I can:
+
+* enable a service to automatically collect and aggregate metrics from my Cloud Kubernetes environment and applications, providing a unified view of all metrics.
+* have metrics visualized in real-time, with interactive charts and graphs, providing insights into the behavior and performance of my Cloud Kubernetes environment and the applications.
+* configure alarms to alert me when specific conditions are met, such as when an application experiences an error.
+* create custom dashboards to view metrics that are most important to me, and to quickly identify areas of concern.
+
+== Iteration: Cluster Autoscaling
+
+As an *End-User* I can:
+
+* enable autoscaling of the cluster so that new worker nodes get added when needed and removed again when not needed anymore.
+* define a lower and upper bound of the amount of worker nodes.
+* specify what kind of worker nodes are part of the autoscaling.
+
+== Iteration: Logs
+
+As an *End-User* I can:
+
+* enable collection of historic logs of my applications running on the cluster.
+* access historic logs of my application running on the cluster.
+* configure retention time of historic logs.
+
+As a *Technical VSHNeer* I can:
+
+* access the logs of the Kubernetes control plane via a user-friendly interface.
+* configure retention time of historic logs.
+
+== Iteration: Service Exposure
+
+As an *End-User* I can:
+
+* expose services to the Internet using the `Ingress` Kubernetes objects.
+* expose services using the Kubernetes service type `LoadBalancer` in order to access it from outside the cluster (for example from the Internet).
+
+== Iteration: TLS Certificate Handling
+
+As an *End-User* I can:
+
+* order and consume TLS certificates which are renewed automatically.
+* use `Ingress` Kubernetes objects with fully automated certificate handling.
+
+== Iteration: Persistent Storage
+
+As an *End-User* I can:
+
+* request and consume RWX (Read-Write-Many) or RWO (Read-Write-Once) storage types.
+* order storage without having to specify a storage class.
+
+== Iteration: Authentication
+
+As an *End-User* and *Technical VSHNeer* I can:
+
+* log in to my cluster through a user-friendly interface
+
+== Iteration: Networking
+
+As a *Technical VSHNeer* I can:
+
+* choose from a pre-defined list of CNI plugins.
```

Comment:
Does the decision to only use Cilium only apply to OpenShift?

Reply:
The idea was to not specify the exact CNI in this document, but in another "pre-defined list of CNI plugins", which then will state that Cilium is the default.